include/cutlass/half.h

```diff
@@ -248,7 +248,7 @@ struct alignas(2) half_t {
 
     if (exp >= -14) {
       // normal fp32 to normal fp16
-      exp = uint16_t(exp + uint16_t(15));
+      exp = exp + 15;
```

Comment:
This looks like it undoes a fix, rather than fixing a warning.

Reply:
This was the only way I could find to shut the warning up. (Not even `exp += 15;` worked.)

No implication of this being the best solution, of course. It did need attention, however, as this was one of the top two sources of warnings in the build.

Reply:
Thanks for explaining!  Now that I look more closely, this makes sense, as `exp` is `int16_t`, a signed integer.  Nevertheless, I would expect the compiler to warn about `exp + 15` being a narrowing conversion (first `exp` promotes to `int`, then the result is `int`, and the assignment narrows to 16 bits).  Would the following line also work?

```suggestion
      exp = static_cast<decltype(exp)>(exp + 15);
```

Use of `decltype(exp)` here follows the Don't Repeat Yourself principle and prevents possible bugs if the type of `exp` later changes above.